apps/web-client/src/lib/stores/replication.ts

```diff
@@ -0,0 +1,182 @@
+import { readable, writable, derived, get } from "svelte/store";
+
+import type { DatabaseInterface } from "@librocco/db";
+
+import type PouchDB from "pouchdb";
+
+const BATCH_SIZE = 5;
```

Comment:
Removing this and using the default batch size sped up things considerably for me. 

I understand you were playing with this to allow for usage in between writes. This is just an observation (also considering the fact that you had planned to halt the app usage until initial sync is done).

Reply:
However...it just came to mind: If we plan to use multiple live syncs (or one-timers), we will probably want to find a fine line in-between in order to service both syncs.

Reply:
I have removed this for now. But on my computer it does make the app very laggy while replication is happening with our large test couchdb